### PR TITLE
chore: cleanup buildkite and project

### PR DESCRIPTION
**Pull request by aeddi**

### CODE REVIEW

File: js/README.md [hunk 1]
```diff
@@ -52,26 +52,36 @@ $ make help
 💡 First, verify you have the [relevant Requirements](#requirements) 😉
 
 ```shell
```
Comment (moul) [inline js/README.md:54]:
```suggestion
```console
```

File: js/README.md [hunk 2]
```diff
@@ -52,26 +52,36 @@ $ make help
...
+$ BERTY_DAEMON_PORT=1338 make.daemon
 
 ## iOS
-## Install generate xcode project
-$ make ios.xcodegen
-## Run on device
-$ IOS_DEVICE=__IOS_DEVICE_NAME__ make ios.run
-## Run multiple devices to connect them to different running bridge ports!
+## Run iOS app in debug mode
+make ios.debug
```
Comment (moul) [inline js/README.md:68]:
```suggestion
$ make ios.debug
```

File: js/README.md [hunk 3]
```diff
@@ -51,27 +51,38 @@ $ make help
...
+## Build and serve UI
 $ make metro.start
 
-## Optional if using one device.
-$ BERTY_DAEMON_PORT=1337 make.bridge
-## Run this with multiple different ports to test with multiple clients.
-## $ BERTY_DAEMON_PORT=1338 make.bridge
+## Optional if using only one device
+## Run this with different ports to test with multiple devices
+$ BERTY_DAEMON_PORT=1337 make.daemon
```
Comment (Jorropo) [inline js/README.md:64]:
```suggestion
$ BERTY_DAEMON_PORT=1337 make daemon.start
```

**Review by Jorropo — CHANGES_REQUESTED** [review]

File: js/README.md (hunk 3 above)
Comment (aeddi) [reply js/README.md:64]: Thanks ;)

**Review by Jorropo — APPROVED** [review]
SGTM

**Review by moul — APPROVED** [review]

**Pull request merged**